server/server.js

```diff
@@ -794,14 +794,18 @@ async function runMediasoupWorkers()
 
 	logger.info('running %d mediasoup Workers...', numWorkers);
 
+	const { logLevel, logTags, rtcMinPort, rtcMaxPort } = config.mediasoup.worker;
+    const portInterval = Math.floor((rtcMaxPort - rtcMinPort) / numWorkers);
+
 	for (let i = 0; i < numWorkers; ++i)
 	{
 		const worker = await mediasoup.createWorker(
 			{
-				logLevel   : config.mediasoup.worker.logLevel,
-				logTags    : config.mediasoup.worker.logTags,
-				rtcMinPort : config.mediasoup.worker.rtcMinPort,
-				rtcMaxPort : config.mediasoup.worker.rtcMaxPort
+				logLevel,
```

Comment:
@vpalmisano I am wondering if  it was by your intention to change the log logLevel and logTags config?

Reply:
@misi but he did not change them I think 
logLevel
is equivalent of 
logLevel : logLevel

Reply:
Sorry I missed out this line:
```
const { logLevel, logTags, rtcMinPort, rtcMaxPort } = config.mediasoup.worker;